.github/workflows/ci-build.yaml

```diff
@@ -128,15 +128,9 @@ jobs:
       - run: make wait
         timeout-minutes: 4
       - run: make ${{matrix.test}} E2E_TIMEOUT=1m STATIC_FILES=false
-      - name: Get wait container logs
+      - name: cat argo.log
         if: ${{ failure() }}
-        run: kubectl get pod -o name | xargs -L 1 kubectl logs -c wait
-      - name: Upload logs
-        if: ${{ failure() }}
-        uses: actions/upload-artifact@v1
-        with:
-          name: ${{matrix.test}}-${{matrix.containerRuntimeExecutor}}
-          path: /tmp/log/argo-e2e
+        run: cat /tmp/log/argo-e2e/argo.log
```

Comment:
rather than have to wait until the whole action fails, make the logs available earlier